composer.json

```diff
@@ -18,7 +18,7 @@
         "psr/http-message": "^1.0",
         "psr/http-factory": "^1.0",
         "psr/container": "^1.0",
-        "ramsey/uuid": "^3.0",
+        "ramsey/uuid": "^4.0",
```

Comment:
```suggestion
        "ramsey/uuid": "^3.0 || ^4.0",
```

Reply:
Let's not force everyone to upgrade to 4, as it mostly contains additions that are not essential for anything in Flow to work. At least not in a minor version - we can discuss limiting to ^4.0 in Flow 7

Reply:
yep agree, makes a lot of sense to avoid conflicting version requirements if you require it in your project too.

Reply:
But wouldn't it be fitting to do it in a new major ?

Reply:
why, we are opening the constraint, if your project relies on functionality of 3.0  that changed in 4.0 you obviously have set that requirement in your composer.json ;)

Reply:
But aren't we blocking other packages that only relies on 4.0, if you own project relies on 3.0?

Reply:
Sure, but that's your problem then and your responsibility to update. I guess we would just support 4.0 from next major then.

Reply:
> But aren't we blocking other packages that only relies on 4.0

We are also blocking those if we only have `^3.0` in our requirement :) Now it's just in your own hands to deal with it